### DISCUSSION TITLE
Nav link Topics and relation to Figma board

### DISCUSSION THREAD

**Discussion by maxwellkahn** [CodeForVegas/cfv-dotorg#7] [Ideas]
In order to update the navbar from a section to a true nav element, I've been comparing the figma board, to the navbar we have now and the stand in section that acted as our initial navbar. 

Since we want to keep things 'sleek' and 'modern' we've decided to cut back on the verbose text blocks of the initial site and opted for those catchy bullet points and divs with small amount of "high impact" wording. 

My question is, what kind of things would you want in our navbar aside from our social links, donation link, and jumps to the "about us" section? If the goal is to keep things light but impactful, is it necessary for us to include more, or can we get away with some bits of scrollable information before we get to the meat of the about us (while maintaining a jump-to button on the nav)?

**Comment by nickjolive** [comment]
I was thinking that a possible aside or sidebar navigation could be a cool idea. We could include a menu button that expands an aside with more links on the page. For the main navbar, Maybe About, Get Involved, Donate? Dan and I have discussed a theory of threes, so maybe we should aim for 3 main points. There is an interesting Youtube video on the subject, [here's the link.](https://youtu.be/uvFmMGwvDXA)

**Reply by maxwellkahn** [reply]
Sure, the CFV stamp would direct us to the homepage without really breaking the rule of three here then. About, Get Involved and Donate seem like great places to start. I'd like to find some design to revamp "about us", as of now it's a lot of scrolling and reading. I would think that by the time a user gets to the end of "about us" they'd lose interest in whatever else might follow (get involved). Something to think about.

**Reply by DanHugoDanHugo** [reply]
It is sort of important to realize that the current content (which anyone reading this in the future will not have seen) was published to satisfy agencies such as Google, lately GitHub, and the IRS, who would be checking to see what this org is about as part of the 1023 filing process for example. Etc. Getting everything on one page *today* was more important than creativity and style.

That said, three pages would be better, eventually announcement pages that could be shareable (that is, fully qualified URLs with opengraph metadata that are shareable in an attractive manner if one were to want to) would be a great idea to get from v0 to v1, at any rate.

The current design (which is not committed anywhere because it was not intended to be more than a placeholder for the sake of filings when it was first published, and it only persists as it is better than a blank page, but it is not ideal by any means) has the content, not the style.

The references to The Figma Board herein illustrate a challenge we will need to overcome in general, linking projects with various tools that are *GitHub-friendly* and those that may not be. This is important not only for collaboration in situ, but for attribution after the fact.

**Reply by nickjolive** [reply]
> Sure, the CFV stamp would direct us to the homepage without really breaking the rule of three here then. About, Get Involved and Donate seem like great places to start. I'd like to find some design to revamp "about us", as of now it's a lot of scrolling and reading. I would think that by the time a user gets to the end of "about us" they'd lose interest in whatever else might follow (get involved). Something to think about.

We did discuss implementing the values section from the Figma design, instead of listing out the values in the Ethos section. That's one example of a way we could spice it up. I also think some subtle additions like scroll animations would be a nice touch.
